src/Adapter/Customer/QueryHandler/GetCustomerForViewingHandler.php

```diff
@@ -386,7 +386,7 @@ private function getCustomerProducts(Customer $customer)
 
             $viewedProducts[] = new ViewedProductInformation(
                 (int) $product->id,
-                Tools::htmlentitiesUTF8($product->name),
+                $product->name,
```

Comment:
Needs to be checked, seems strange to remove a sanitization step 🤔

Reply:
It's intentional, I asked to be sure in https://github.com/PrestaShop/PrestaShop/pull/17065#issuecomment-573575948